util/cc2file.go.tmpl

```diff
@@ -0,0 +1,34 @@
+{{/*
+	This command sends your CC(s) code in a text file, rather than "plain" Discord messages, preserving Tabs, markdown, etc. 
+	You can especify more than one ID or trigger, and bot will generate one file per input parameter. If none are provided, output will be CC list
+	
+	Usage: `-cc2file <ID or Trigger> [ID or Trigger] [ID or Trigger]...`. For example: `-cc2file 10 "te st" 23`
+	Recommended trigger: Command trigger with trigger `cc2file`.
+	Configuration variables: None
+*/}}
+
+
+{{range .CmdArgs}}
+	{{- $exec_cc_Answer:= exec "cc" .}}
+	{{- $answerLines:= split $exec_cc_Answer "\n"}}
+
+	{{- $extractedCode:= slice $answerLines 2 (sub (len $answerLines) 1)}}
+	{{- $firstLine:= index $answerLines 0}}
+
+	{{- if reFind "here is a list of" $firstLine}}
+		{{- sendMessage nil (printf "❌ CC `%s` not found" .)}}
+	{{- else}}
+		{{- sendMessage nil (complexMessage
+			"content" $firstLine
+			"file" (joinStr "\n" $extractedCode)
+		)}}
+	{{- end}}
+{{else}}
+	{{with (exec "cc")}}
+		{{if gt (len .) 2000}}
+			Use `-cc` command to generate CC list. You shouldn't need to do this, I have to fix that
```

Comment:
Are you planning on resolving this issue?

Reply:
At least not yet. Maybe in one week or so, when I have free time

Reply:
Done! New `sendLongMessage` template